llama/convert.py

```diff
@@ -32,11 +33,12 @@ def map_torch_to_mlx(key, value):
     elif "rope" in key:
         return None, None
 
-    return key, value.numpy()
+    return key, value.numpy() if value.dtype != torch.bfloat16 else value.to(torch.float32).numpy()
```

Comment:
Should we use float16`value.to(torch.float16).numpy()` to avoid increasing the size?

Reply:
With float16
```
  13476925163 Dec  6  consolidated.00.pth
  13476918398 Dec  6  mlx_llama_weights.npz
```

Reply:
I agree it's a good idea to use `float16` to save the weights. It will be half the size which is nice.  I did the same in the Mistral example, it works fine to store the weights in fp16.

Reply:
Changed to `torch.float16`
```
13476918398 Dec  7 07:49 mlx_llama-2-7b.npz
```